src/main/java/com/example/wdmsystem/employee/system/authentication/AuthenticationController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.wdmsystem.employee.system.authentication;
+
+import com.example.wdmsystem.util.JwtUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    @Autowired
+    private AuthenticationManager authenticationManager;
+
+    @Autowired
+    private JwtUtil jwtUtil;
+
+    @Autowired
+    private CustomUserDetailsService userDetailsService;
```

Comment:
Is it not possible to put these in a constructor? As far as I'm aware @Autowired is old and it may be "correct" practice to use a constructor.

Reply:
Alright, will change it in my next branch